content/html/concepts/attributes/terms/title/title.md

```diff
@@ -0,0 +1,49 @@
+---
+Title: 'title'
+Description: 'Contains and can display the specified information about the element it belongs to.'
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'Attributes'
+  - 'Elements'
+  - 'Tags'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The **`title`** global attribute contains and can display the specified information about the element it belongs to.
+
+> **Note:** Although the `title` attribute can display additional information on the screen, it does not work with touch screens and keyboard-only navigation. Using the `title` attribute is especially important for assistive technologies to help provide a user-friendly browsing experience.
+
+## Syntax
+
+```pseudo
+<opening-tag title="text"> content </closing-tag>
+```
+
+The `title` attribute can be used with any HTML [`element`](https://www.codecademy.com/resources/docs/html/elements) by adding the attribute to the opening tag. The `text` value of the attribute can be any string, including empty string (`""`) and multi-line text. This `text` will appear next to the element as a tooltip text (hint) while hovering over the element it is used with.
+
+For nested elements, the `title` attribute will be inherited from the parent element if the child element has no `title` attribute.
+
+### Special Use Cases
+
+While any element can have a `title` attribute, it provides a special function for some elements. When the `title` attribute is used with:
+
+- the [`<abbr>`](https://www.codecademy.com/resources/docs/html/elements/abbr) element, the `title` must contain a description or expansion of the abbreviation.
```

Comment:
```suggestion
- [`<abbr>`](https://www.codecademy.com/resources/docs/html/elements/abbr): The `title` must contain a description or expansion of the abbreviation.
```

Reply:
I think it's a little cleaner to start with the link name, then colon, description